test/findCSSAssetPaths-test.js

```diff
@@ -17,6 +17,11 @@ it('finds images', () => {
   expect(subject()).toEqual([{ assetPath: '/1x1.png', resolvePath: '/1x1.png' }]);
 });
 
+it('strips query strings', () => {
+  css = '.body { background-image: url("/1x1.png?pizza=yum");';
+  expect(subject()).toEqual([{ assetPath: '/1x1.png', resolvePath: '/1x1.png' }]);
```

Comment:
@trotzig I wasn't entirely sure if the query string should be stripped for both assetPath and resolvePath

Reply:
You did the right thing.